### DISCUSSION TITLE
ETPRO Suricata 7 Rule-set

### DISCUSSION THREAD

**Discussion by stondino00** [Security-Onion-Solutions/securityonion#13243] [2.4] [answered]
### Version

2.4.70

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

192

### Storage for /

1TB

### Storage for /nsm

20TB

### Network Traffic Collection

span port

### Network Traffic Speeds

more than 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

ETPRO recently this week enabled rule sets for Suricata 7.  Is this a supported change in SO that we should be making to get that newer rule set?  If so, how do we make this change?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by stondino00** [comment]
Any thoughts on this?

**Reply by TOoSmOotH** [reply, marked as answer]
We are testing internally and will add support in the next release.

**Reply by stondino00** [reply]
Ok sounds great.  Thanks so much.
